packages/textile/src/users.spec.ts

```diff
@@ -107,7 +109,8 @@ describe('Users...', () => {
       }
       // All good
       const id = ThreadID.fromRandom()
-      await db.newDB(id, ctx.withThreadName('foo'))
+      db.context.withThreadName('foo')
```

Comment:
once https://github.com/textileio/js-threads/pull/263 lands, we can drop this

Reply:
fails